src/export/bops/index.ts

```diff
@@ -159,6 +158,29 @@ export function formatProposalDetails({
     );
     if (!isTypeForBopsPayload(crumb.type) || !validKey) continue;
 
+    const metadata = (() => {
+      const metadata: QuestionMetaData = {};
+      if (hasSections) {
+        metadata.section_name = currentSectionName;
+      }
+      if (crumb.autoAnswered) metadata.auto_answered = true;
+      if (node.data?.policyRef) {
+        const policyRefs = parsePolicyRefs(node.data.policyRef as string);
+        if (policyRefs.length) metadata.policy_refs = policyRefs;
+      }
+      return metadata;
+    })();
+
+    if (isSchemaType(crumb.type)) {
```

Comment:
**Why is this not handled in the normal per-component pattern?**

Here's how the works currently - 
 - Iterate over answers in breadcrumbs 
 - Find corresponding node in flow, to get the question put to users
 
This pattern doesn't work for more complex, schema driven, components. This is because there's not a 1-to-1 relationship between questions and answers per-node as in the previously established pattern.

We need to break out of this loop to handle schema driven components so that we can create multiple questions and responses per-node.